typesense/api/openapi.yml

```diff
@@ -1268,6 +1268,9 @@ components:
           type: boolean
           example: false
           default: false
+        index:
+          type: boolean
+          example: false
```

Comment:
I think it is better to add `default: true` (all fields are indexed by default) so that the generated type is a `bool` not `*bool`

Reply:
It is a `*bool` because it is not required. But I do not think it should be required. The `default: true` does not change anything.

Reply:
ok, I will merge this PR once I figure out why CI builds stop starting automatically in this repository.

Reply:
Thanks a lot 😃 Would you please also do a release after the merge? Or do you plan a bigger release in the near future?

Reply:
Yeah, okay, I will create a v0.3.0 release after the merge. I plan to add Multi Search support in the near future but it may be done already in the v0.4.0 release.